### ISSUE TITLE
Loss: nan

### ISSUE THREAD

**Issue by haofusheng** [delijati/pytorch-siamese#3]
I tried to run this train_mnist.py, but came across Loss: nan.

Also, it is not helpful to reduce the learning rate.

Do you have a way to make the network converge?

Many thanks!

**Comment by gsygsy96** [comment]
EMMM... I met the same question. Maybe any useful tip can provided?

**Comment by gsygsy96** [comment]
I found answer in closed issue. python3 will be ok.

**Comment by nihaoxiaoli** [comment]
Do you know the reason? @shanYanGuan

**Comment by nihaoxiaoli** [comment]
I got it. If we use python2, we should use ```x0_data.append(data[z1] / 255.0) ``` in preprocessing phase.

**Comment by delijati** [comment]
Fixed. Damn `python2` ;)
